scss/_layouts_application-panels.scss

```diff
@@ -65,6 +113,11 @@
     padding-top: $spv--large;
```

Comment:
This already seems bit too large (buttons are positioned lower than header text) so maybe reducing it down to `$spv--medium` would help.